package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -1144,27 +1143,49 @@ dnsmasq_start()
 
 	xappend "--dhcp-broadcast=tag:needs-broadcast"
 
+	# Set up daemon config paths
+	_commonconfdir="/tmp/dnsmasq.d"
```

Comment:
I'd move this up above line 20 and make it CAPS.

Reply:
Reasonable. Thanks. I was hoping to avoid 'uglification' by screaming caps :)